### DISCUSSION TITLE
How can I find the package of org.apache.skywalking.apm.network.logging and common

### DISCUSSION THREAD

**Discussion by yqlboss** [apache/skywalking#7476] [Q&A]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?
- I want see something,but it seems that some classes and packages cannot be found.
`org.apache.skywalking.apm.network.common`
`org.apache.skywalking.apm.network.logging.v3`

___
### Bug
- Which version of SkyWalking, OS, and JRE?

- Which company or project?

- What happened?
If possible, provide a way to reproduce the error. e.g. demo application, component version.

___
### Requirement or improvement
- Please describe your requirements or improvement suggestions.

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/main/latest/en/guides/how-to-build/#how-to-build-a-project
